exonum/src/events/noise/wrapper.rs

```diff
@@ -126,9 +126,9 @@ impl NoiseWrapper {
 
         msg.chunks(NOISE_MAX_MESSAGE_LENGTH - TAG_LENGTH)
             .for_each(|msg| {
-                let (written_bytes, written) = self.write(msg).unwrap();
+                let written = self.write(msg).unwrap();
```

Comment:
- I'd add an `assert` or `debug_assert` that the length of `written` is as expected (`msg.len() + TAG_LENGTH`, if I understand correctly).
- Why is `unwrap()` OK here?

Reply:
`unwrap` is not OK, I'll remove it.